src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -10,12 +15,12 @@ case class AttributeRecord(id: Long,
                            name: String,
                            valueString: Option[String],
                            valueNumber: Option[Double],
-                           valueDate: Option[Timestamp],
```

Comment:
because there is no AttributeDate case class which means we don't actually support it now.